### ISSUE TITLE
Since fediverse url's are a bit confusing can we have a standard "copy post or community url" that can be used as an anchor on all instances to find something specific?

### ISSUE THREAD

**Issue by Leopere** [LemmyNet/lemmy-ui#1901]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

I know that I can just take the time to learn the pattern of the url's similarly to how it was on the dead place but ultimately I would really love if we could have some kind of "copy post/thread id" that would help with search function.  One of the things I really need to be happy with the transition to Lemmy is I need some way to record/save useful stuff right now it feels like I first need to find and subscribe to a community first before I can "Save" anything and that's just too many steps for something that is supremely useful for helping me remember certain things that I want to remember.

**Comment by brihuang99** [comment]
I agree, trying to memorize the community name + the instance name and then tracking down said post is a lot of work. It'd be nice to have a unique identifier of some sort for a specific post

**Comment by Leopere** [comment]
one of the functions that I really enjoyed about previous now redundant centralized social media platforms was the fact that we could simply review old posts that were saved to our accounts.  In this case i am not really entirely sure outside of adding a bookmark somehow through means of using tools like either Shiori or WebsiteShot how I would archive this information on Lemmy.  Perhaps I'm imagining some need for a sort of extra app of some kind like a bookmarklet or something (I think it needs to be done somewhere within the lemmy UI frankly) I know it would be hard with cookies becoming more and more sandboxed in modern browser implementations but still would be really quite nice to have.

**Comment by dessalines** [comment]
We already have the fedilink as the share link.
